src/routes/donations.js

```diff
@@ -171,13 +178,19 @@ function _upsertDonorInsertDonation({
     returning: true
   }).then(([donor, created]) => {
     return new Promise(res => {
-      return db.Donation.create({
-        ...donation,
-        donorId: donor.id
-      }, {
-        transaction
-      }).then(() =>
-        res([...previousResult, { ...donor.toJSON(), __isNew: created }])
+      return db.Donation.create(
+        {
+          ...donation,
+          donorId: donor.id
+        },
+        {
+          transaction
+        }
+      ).then(() =>
+        res([
+          ...previousResult,
+          { ...donor.toJSON(), __isNew: created, ...donation }
```

Comment:
Where did the `...donation` part come from?

Reply:
we need donation amount, donation date and to derive donation count from ...donation